instances.tf

```diff
@@ -116,16 +116,16 @@ resource "aws_security_group" "tableau" {
     ]
   }
 
-  ingress {
-    from_port = 1024
-    to_port   = 65535
-    protocol  = "TCP"
-
-    cidr_blocks = [
-      var.tableau_subnet_cidr_block,
-      var.vpc_subnet_cidr_block,
-    ]
-  }
+  #ingress {
```

Comment:
If this port is in question then do you think we need to investigate why and who used this?
Also, if it is concluded its not needed then is there any reason to have commented code rather than removing it?

Reply:
@hemantgoswami 
This port has been removed by Mike from Console under this ticket - https://jira.dsa.homeoffice.gov.uk/browse/PM-75595,
AWS Console CloudTrail - https://eu-west-2.console.aws.amazon.com/cloudtrailv2/home?region=eu-west-2#/events/fab4ff1e-575d-4077-868c-60722697e21d

This ticket is dealt by CIT team at the moment, so I have commented for now to deploy our code from [dq-tf-consolidated-schedule-pipeline](https://gitlab.digital.homeoffice.gov.uk/dacc-dq/dq-tf-consolidated-schedule-pipeline)